runtime/astar/src/lib.rs

```diff
@@ -460,6 +460,7 @@ impl pallet_dapp_staking::Config for Runtime {
     type MinimumStakeAmount = MinimumStakingAmount;
     type NumberOfTiers = ConstU32<4>;
     type RankingEnabled = ConstBool<true>;
+    type MaxBonusSafeMovesPerPeriod = ConstU8<0>;
```

Comment:
General question for all runtimes - did you try to test the migration extrinsic yet?

Reply:
Yes, I've tested via **try-runtime** CLI & **chopsticks**. We should be able to update up to **399** entries per call.